### DISCUSSION TITLE
Font size and items per page on search file results

### DISCUSSION THREAD

**Discussion by sonix-github** [koreader/koreader#9958] [Q&A] [answered]
Hello,

I am using KOReaders built-in file search function frequently and the font in table with results is quite small. Unfortunately I have not found setting where can I choose font size nor items count per page.

Could someone advice if the setting is somewhere "hidden" in menu, please?

Thanks

![file_search_result_list_font](https://user-images.githubusercontent.com/29783895/209563007-97e0e519-b08e-4468-8b96-d95459caa23e.png)

**Comment by hius07** [comment, marked as answer]
We have a common setting which affects all lists:
menu Filemanager - Settings - Classic mode settings - Item font size

**Reply by sonix-github** [reply]
> We have a common setting which affects all lists: menu Filemanager - Settings - Classic mode settings - Item font size

Thanks @hius07 for the hint. I did not know that this settings affects also search result list.
